.golangci.yml

```diff
@@ -0,0 +1,63 @@
+version: "2"
+formatters:
+  enable:
+    - gofmt
+    - goimports
+linters:
+  default: none
+  enable:
+    - bodyclose
+    - gocritic
+    - gosec
+    - govet
+    - nestif
+    - nlreturn
+    - revive
+    - rowserrcheck
+    - staticcheck
+  settings:
+    gosec:
+      excludes:
+        - G104
+        - G115
```

Comment:
Why make this difference for `G115` compared to the SDA repo?

Reply:
G115 should be in the default exclude list because max(int32) < max(uint64).
This should be added to the SDA repo as well.